packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/UnstakeModal/UnstakeModal.tsx

```diff
@@ -0,0 +1,32 @@
+import styled from 'styled-components';
+import { useSelector } from 'src/hooks/suite';
+import { Modal, Translation } from 'src/components/suite';
+import { UnstakeModalContent } from './UnstakeModalContent';
+
+const StyledModal = styled(Modal)`
+    width: 512px;
```

Comment:
Not sure it causes a problem since the base component has min and max widths, which are prioritised first. Moreover, there are quite a lot of other modal windows in the project that use this very same approach, e.g. `TorLoadingScreen`, `ReconnectDevicePrompt`, `SkipStepConfirmation` etc. Providing a screenshot from Chrome for mobile devices. Let me know if there is a specific problem which I’m missing.

<img width="452" alt="Знімок екрана 2024-01-04 о 13 53 33" src="https://github.com/everstake/trezor-suite/assets/42133844/26445a11-6a46-41bf-8fc9-723931be672a">

Reply:
Oh, you right, sorry